src/devices/oregon_scientific.c

```diff
@@ -142,6 +142,11 @@ static unsigned long long cm180i_total(uint8_t const *msg)
     return val;
 }
 
+static uint8_t swap_nibbles(unsigned char byte)
```

Comment:
other type should also be `uint8_t`, we don't use `unsigned char`

Reply:
Adjusted.